core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -805,4 +805,73 @@ class PlaintextConsumerTest extends BaseConsumerTest {
 
     consumer2.close()
   }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testSubscribeAndCommitSync(quorum: String, groupProtocol: String): Unit = {
```

Comment:
We have split the consumer tests into separate files grouped by feature, and there is now one `PlaintextConsumerCommitTest`, I would expect this test should go there.

Reply:
I think this is an error from rebase. so this should be removed from the PR.  Thanks for catching this.